src/chart/sankey/sankeyLayout.js

```diff
@@ -28,6 +28,8 @@ export default function (ecModel, api, payload) {
         var nodeWidth = seriesModel.get('nodeWidth');
         var nodeGap = seriesModel.get('nodeGap');
 
+        var mode = seriesModel.get('mode') || 'normal';
```

Comment:
You should probably set default mode in `SankeySeries.js` rather than using `|| 'normal'`.

Reply:
done @Ovilia